OpenRA.Game/Network/ReplayRecorder.cs

```diff
@@ -69,6 +69,9 @@ void StartSavingReplay(byte[] initialContent)
 
 		public void Receive(int clientID, byte[] data)
 		{
+			if (disposed)
+				return;
```

Comment:
What I'm getting at is: we should fix it so nobody calls `Receive` after we've already disposed `ReplayRecorder`. We *should* crash if `ReplayRecorder` is getting new stuff after being disposed, because that likely indicates a bug.